src/core/importType.js

```diff
@@ -29,29 +34,11 @@ export function isExternalModule(name, settings, path, context) {
   if (arguments.length < 4) {
     throw new TypeError('isExternalModule: name, settings, path, and context are all required');
   }
-  return (isModule(name) || isScoped(name)) && isExternalPath(name, settings, path, getContextPackagePath(context));
+  return (isModule(name) || isScoped(name)) && (isExternalPath(path, context) || isExternalLookingName(name));
```

Comment:
it seems like `isExternalModule` is missing the `isInternalPath` logic that's in `typeTest`. Should these be in sync? or perhaps, should this function use `typeTest`'s result instead?

(same question about `isExternalModuleMain`)

Reply:
Great catch! Yeah just updated to use `typeTest`, that seems like a good idea

Reply:
I'm also realizing that at least in theory to be as backwards compatible as possible the `settings` argument should get used if provided. So I just updated that, too